packages/chrome/src/styled/sheet/StyledSheetDescription.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { math } from 'polished';
+
+const COMPONENT_ID = 'chrome.sheet_description';
+
+const fontStyles = (props: ThemeProps<DefaultTheme>) => {
+  return css`
+    line-height: ${math(`${props.theme.lineHeights.sm} - 1px`)};
+    font-size: ${math(`${props.theme.fontSizes.sm} - 1px`)};
```

Comment:
These values are off-system (and off design). Use the [getLineHeight](https://garden.zendesk.com/components/utilities#getlineheight) function for unitless line height. Also, the design-specified color is missing.

Please reference https://github.com/zendeskgarden/react-components/blob/main/packages/tags/src/styled/StyledTag.ts for an example of expected code format.

Reply:
Good catch on size being off design, will update!